<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>357</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>522</width>
    <height>357</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>522</width>
    <height>357</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="fileTree_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>501</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="updateBtn_">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>250</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Code</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipField_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>RPi IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="ipHelp_">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>257</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/resources/images/info.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
